src/components/dashboard/SendTokens/Bitcoin/FormContainer.jsx

```diff
@@ -4,11 +4,15 @@
  */
 
 import React from 'react'
+import { connect } from 'react-redux'
+
 import Form from './Form'
-import FormContainer from '../BitcoinLikeBockchain/FormContainer'
+import FormContainer, { mapStateToProps, mapDispatchToProps } from '../BitcoinLikeBockchain/FormContainer'
+
+const FormReduxContainer = connect(mapStateToProps, mapDispatchToProps)(FormContainer)
 
-const bitcoinFormContainer = (props) => (
-  <FormContainer {...props} form={Form} />
+const BitcoinFormContainer = (props) => (
+  <FormReduxContainer {...props} form={Form} />
```

Comment:
Why we need BitcoinLikeBlockchain form container and Form if we use it as is for BTC, LTC, BCC and there is only a few small differences for DASH? Do we really need such "optimization" for several lines?

Reply:
There is no optimization. This is the shortest way to make behavior of the Dash form different from the behavior of other bitcoin-like forms